playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every single developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
```

Comment:
I'm sorry if I hurt your feelings in any way with this statement :) 
Let's meet in the middle here. I agree this can be improved, but I think the first statement should be bold as it is. Too many "should be's"  can soften up a rulebook too much. It is a rulebook after all.
Also, it is a nice way of writing, to start a paragraph with a bold statement. :)
It makes reader more interested in reading further.

Reply:
It's okay, I know how to recover from it 😄 

Actually, it is a playbook and playbooks are considered to be a set of suggestions, rather than a set of strict rules.

Depends on who is reading 😆 

In the end, it doesn't even matter what do we call it, nor in which tone it has been written, if you don't feel like changing it, fine by me.
But bear in mind that this document can and can not be respected while performing or submitting a PR. ✌️

Reply:
I will be accepting a mix between @ibrcko and @markopaden versions.
It **is** the developers responsibility to review code, and the developer should know why he is given this responsibility, what are the benefits and how to perform the task well.

And as with any good code review, it is always important to keep the discussion open and professional. This will greatly increase the chances of reaching a globally optimal solution :)